Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -197,6 +197,7 @@ final class NetworkProtectionConnectionTester {
             self?.timer = nil
         }
 
+        isRunning = true
```

Comment:
This was causing the connection tester to never call its callback block, which is what triggers rekeying and the DAU pixel. This var is set to true when `start` is first called, but then after that the class calls the `stop` function to kill any old timers that might be occurring, which then sets `isRunning` back to false.

Reply:
I saw that message in tests yesterday.